### DISCUSSION TITLE
Welcome to ByteWalk Discussions!

### DISCUSSION THREAD

**Discussion by shubham1710** [shubham1710/ByteWalk#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prafulrathore** [comment]
hey, could you please share the database schema design of Bytewalk repo

**Reply by shubham1710** [reply]
Django works with Relational Database. In development, I am using the default database (SqLite) and in production, I am using Postgres database which comes with Heroku.

**Comment by prafulrathore** [comment]
I know django works on dbsqlite by default,but  i need to understand the
database design of this site which u had made to understand in better
manner ,could you send me pic of database design in table format it will be
good for me??

**Reply by shubham1710** [reply]
You can see the Models in this Django app to know how the database would be structured.

**Reply by bukscom** [reply]
can you share its output link? or output screenshots?

**Reply by shubham1710** [reply]
> can you share its output link? or output screenshots?

It was live until a few months back but my Google Cloud Storage trial expired so it went down. Anyways, you can run it in your local machine by using your credentials for Cloud and can see the output.
